actions.go

```diff
@@ -405,6 +405,11 @@ func (c *Action) GetIDToken(ctx context.Context, audience string) (string, error
 	return tokenResp.Value, nil
 }
 
+// Getenv retrieves the value of the environment variable named by the key.
```

Comment:
Can you document that it uses the internal Getenv function?

Reply:
Done, added the second line to this comment